docs/scripts/document-metadata.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+DOCS_DIR="docs/*.md"
+
+DOCUMENTS=$(git ls-files $DOCS_DIR)
+
+for MD in $DOCUMENTS; do
+    echo "updating: $MD"
+
+    PLACEHOLDER="{{last-updated-at}}"
+    REPLACEMENT=$(git --no-pager log -1 --pretty=format:'%ad' --date=local $MD)
+    sed -i "s/$PLACEHOLDER/$REPLACEMENT/g" $MD
+    
+    PLACEHOLDER="{{relative-time-since}}"
+    REPLACEMENT=$(git --no-pager log -1 --pretty=format:'%cr' $MD)
+    sed -i "s/$PLACEHOLDER/$REPLACEMENT/g" $MD
+
+    PLACEHOLDER="{{commit-hash}}"
+    REPLACEMENT=$(git --no-pager log -1 --pretty=format:'%h' $MD)
+    sed -i "s/$PLACEHOLDER/$REPLACEMENT/g" $MD
+
+    PLACEHOLDER="{{last-translated-at}}"
+    REPLACEMENT=$(git --no-pager log -1 --pretty=format:'%ad' --date=local docs/i18n)
```

Comment:
Seeing this now, this is a bit more complicated actually :thinking: 
This will yield the date where _any_ translation was written. But we want that to be specific to that page. 

Which makes me wonder for a minute: we have possibly many translations per page (fr, jp, ...). So we have 2 options I believe:

- 1. Indicate when each translation was last translated. On the one hand, this is handy to see which translation are lagging behind from the reference source. Yet on the other hand, it can become cumbersome / noisy once we start adding  more languages.

- 2. Instead of indicating the `last-translated-at` on the reference source (i.e. `en`), we instead put a warning on each translated page if the source was modified AFTER the translation. Something a bit visible that gives readers a clear warning "This translation may not be up-to-date, the source was changed X days ago!". 

I think option (2) is also easier to handle / implement with that bash script approach, as we can simply leave `last-translated-at` blank for files outsides of `i18n` and, for thos in `i18n`, look for the original under the same path and indicates both date to the component. And in the component, either: 

- display only the "last updated at" relative time; or
- display the "last updated at" relative time AND a warning if it is before the "last updated at" of the source.

Reply:
New looking style
<img width="575" alt="image" src="https://user-images.githubusercontent.com/14299604/184735619-3695da58-8882-4955-bf1e-e50035cca01e.png">
wdyt?